network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -2,6 +2,9 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block block_content %}
+    <div class="tw-px-2 tw-py-12 tw-w-full">
+        <h2 class="tw-h3-heading">{{ self.heading }}</h2>
+    </div>
```

Comment:
Since the heading is optional, I think we should wrap the `div` in an `if` block, especially since it adds padding.

Reply:
:+1: